### DISCUSSION TITLE
[Question] Client Version World of Warcraft: Shadowlands

### DISCUSSION THREAD

**Discussion by xBarselonx** [TrinityCore/TrinityCore#28568] [Q&A] [answered]
What is the minimum version of the World of Warcraft: Shadowlands game client?

**Comment by Aokromes** [comment, marked as answer]
there is no minimum, we only support a single version
TrinityCore Open Source MMO Framework (master = 9.2.7.45745

**Reply by xBarselonx** [reply]
Thanks
